### ISSUE TITLE
(octave.portable) Adopt "octave" package to synchronize them

### ISSUE THREAD

**Issue by ComFreek** [chocolatey-community/chocolatey-coreteampackages#1131]
## Expected Behavior
[octave][1] and [octave.portable][2] can be expected to be synchronized, i.e. reflect the same software incl. architecture bitness and most importantly the same version number.

## Current Behavior
[octave][1] is heavily outdated with version 3.6.4 compared to [octave.portable][2] with version 4.4.1.
(**Edit:** ^ mistakenly confused octave and octave.portable)

## Possible Solution
Adopt the [octave][1] package, probably by reusing most parts of [octave.portable's][2] install script.

## Context
I wanted to install the [octave][1] package, which then in turn installed an outdated version of [octave.portable][2].

[1]: https://chocolatey.org/packages/octave
[2]: https://chocolatey.org/packages/octave.portable

**Comment by AdmiringWorm** [comment]
I think you've mixed the octave.portable and octave packages, it's  the octave.portable that is at v 4.4.1, and the octave package at 3.6.4.

I don't get how you got the octave to install the octave.portable version though, as it doesn't have a dependency on that package.

It is also set up to be a package that uses the installer, and not the archives (like the portable edition uses).

So even if we would take over the maintenance of the octave package, this would continue to use the installer (along with a different package called octave.install which would contain the meat of the installation).

**Comment by ComFreek** [comment]
> I think you've mixed the octave.portable and octave packages, it's the octave.portable that is at v 4.4.1, and the octave package at 3.6.4.

Indeed, that's now corrected in the OP.

> I don't get how you got the octave to install the octave.portable version though, as it doesn't have a dependency on that package.

I've just checked that again by inspecting the nupkg... that's strange. I remember Chocolatey asking me whether I wished to uninstall octave.portable upon uninstallation of octave.

Maybe I can arrange a PR next weekend. Just to be sure, the new octave package should be empty and just take a dependency on the (then newly created) octave.install?

**Comment by AdmiringWorm** [comment]
Sure, feel free to do that.

However, I do believe we should try contacting the existing maintainer of the octave package as well (although I don't believe we'll hear anything back).

But we even for packages that are being migrated to this repo, the triage process needs to be done first as well.

**Comment by ComFreek** [comment]
@stale Yes, this is still relevant: The [octave](https://chocolatey.org/packages/octave) package is heavily outdated contrary to [octave.portable](https://chocolatey.org/packages/octave.portable) being up-to-date. Previous comments still apply, someone needs to adapt that package.

**Comment by AdmiringWorm** [comment]
It is unfortunately not relevant, as the octave package is not part of this repository (I understand the want to have it updated together with the portable edition), please follow the [triage process](https://chocolatey.org/docs/package-triage-process#the-triage-process). After that process is done, then we can get this added to the repository.

**Comment by ComFreek** [comment]
I've just seen somebody has taken on the task: https://github.com/chocolatey-community/chocolatey-package-requests/issues/487 🎆
